guides/posts/AI-apis/post.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Top 5 Artificial Intelligence APIs"
+description: "Top 5 Artificial Intelligence APIs to build excellent full-stack applications."
+slug: "top-artificial-intelligence-apis"
```

Comment:
The [source](https://github.com/RapidAPI/DevRel-Stack-Data/pull/79/files#diff-013dbcae57c03571c8d68fc66f69a6a154bdfd5e2bee0a7ef91c7d1da11821eaR6) and the slug should be the same.